moonwalking/blocks/base.py

```diff
@@ -50,8 +50,21 @@ def validate_addr(self, addr: str):
     async def create_wallet(self) -> Tuple[str, str]:
         raise NotImplementedError
 
-    @abstractmethod
     async def send_money(self, priv: str, addrs: List[Tuple[str, D]]) -> str:
+        tx = await self.build_tx(priv, addrs)
+        signed = self.sign_tx(priv, tx)
+        return await self.broadcast_tx(signed)
+
+    @abstractmethod
+    async def build_tx(self, priv: str, addrs: List[Tuple[str, D]]):
+        raise NotImplementedError
```

Comment:
Don't think we need to `raise` and use `@abstractmethod`

Reply:
Agreed, and ideally we'd have a docstring rather than `pass`...